### PR TITLE
Add 'm' 4-byte MIDI type tag

### PR DESCRIPTION
**Pull request by levaitamas**
Refs #42.
Successfully tested with external software (pyo). 

Comments welcome.

### CODE REVIEW

File: pythonosc/test/parsing/test_osc_types.py [hunk 1]
```diff
@@ -123,6 +123,45 @@ def test_datagram_too_short(self):
     self.assertRaises(osc_types.ParseError, osc_types.get_rgba, dgram, 2)
 
 
+class TestMidi(unittest.TestCase):  # TODO
```
Comment (attwad) [inline pythonosc/test/parsing/test_osc_types.py]: remove TODO statement? seems done to me :)

**Comment by attwad** [comment]
Thanks for the PR! This looks good, just a few nits, then I'll merge.

**Comment by levaitamas** [comment]
Thanks! 
I have addressed your comments with the last 2 commits.

**Comment by attwad** [comment]
Thanks! merging~

**Pull request merged**